editor-blender/core/actions/state/load/music.py

```diff
@@ -25,11 +25,7 @@ def setup_music(assets_load: dict[str, Any]):
         "music", filepath=music_filepath, channel=1, frame_start=0
     )
 
-    # set frame range
-    bpy.context.scene.frame_start = 0
-    bpy.context.scene.frame_end = bpy.context.scene.sequence_editor.sequences[
-        0
-    ].frame_duration
```

Comment:
Keep these, the users should be able to preview the time interval they are selecting, if you remove this part the frame interval will be locked to 1~250 before partial load.